mozilla/fx-private-relay-add-on
# 2.6.3 Release

- Merge in latest l10n strings
- add announcement panel
- Merge in latest l10n strings
- Merge in latest l10n strings
- Merge in latest l10n strings
- Merge in latest l10n strings
- Merge in latest l10n strings
- Merge in latest l10n strings
- Merge in latest l10n strings
- Merge in latest l10n strings
- Merge in latest l10n strings
- Merge in latest l10n strings
- Merge in latest l10n strings
- Merge in latest l10n strings
- Ensure the mask filter match count is consistent
- MPP-2534 - Do not display negative masks in popup
- more accurate variable name
- Remove references to `phones` and `bundle` flags
- Make Chrome extension styles override website ones
- Ensure website styles are overridden
- fix layout issue with image
- rearrange panels
- remove has_phone and has_vpn
- updtae premium image
- Merge in latest l10n strings
- Merge in latest l10n strings
- Merge in latest l10n strings
- Merge in latest l10n strings
- Merge in latest l10n strings
- Merge in latest l10n strings
- Merge in latest l10n strings
- Merge in latest l10n strings
- Merge in latest l10n strings
- Merge in latest l10n strings
